Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -1410,7 +1410,7 @@ For more information, ensure that you have the fix `Pull Request <https://github
 Limitations
 ###########
 
-    * Cilium's eBPF kube-proxy replacement currently cannot be used with :ref:`gsg_encryption`.
```

Comment:
Could you update the title of the first commit (and of the PR) to something more descriptive? Maybe `docs: Update KPR limitations wrt WireGuard`?